src/components/wrappers/FormWrapper.tsx

```diff
@@ -0,0 +1,24 @@
+import { useScreenOptions } from '@baca/hooks'
+import { FC, PropsWithChildren } from 'react'
+import { StyleSheet } from 'react-native'
+
+import { KeyboardAwareScrollView } from '../KeyboardAwareScrollView'
+
+export const FormWrapper: FC<PropsWithChildren> = ({ children }) => {
+  useScreenOptions({ headerShown: false })
```

Comment:
**Suggestion:** 
Maybe this option should come with props 🤔 

It's very common that we will don't want to hide header.